vocs/docs/pages/rpc-providers/setting-up-a-provider.md

```diff
@@ -1,28 +1,27 @@
-## Setting up a Provider
+## RPC Provider
 
 A [`Provider`](https://docs.rs/alloy/latest/alloy/providers/trait.Provider.html) is an abstraction of a connection to the Ethereum network, providing a concise, consistent interface to standard Ethereum node functionality.
 
-### Builder
+### Provider Builder
 
 The correct way of creating a [`Provider`](https://docs.rs/alloy/latest/alloy/providers/trait.Provider.html) is through the [`ProviderBuilder`](https://docs.rs/alloy/latest/alloy/providers/struct.ProviderBuilder.html), a [builder](https://rust-unofficial.github.io/patterns/patterns/creational/builder.html).
```

Comment:
>a [builder](https://rust-unofficial.github.io/patterns/patterns/creational/builder.html).

this sounds a bit awkward, would rather say smth like "which follows the builder pattern" for simple initialisation